COMPILING.md

```diff
@@ -89,8 +89,8 @@ Most users will probably want to use this method:
 
 ### Compiling only
 
-1. Create a folder under `\windows` called ASIOSDK2
-1. Download the [ASIOSDK](https://www.steinberg.net/asiosdk), open the top level folder in the .zip file and copy the contents into `[\path\to\jamulus\source]\windows\ASIOSDK2` so that, e.g., the folder `[\path\to\jamulus\source]\windows\ASIOSDK2\common` exists
+1. Create a folder under `\libs` called ASIOSDK2
+1. Download the [ASIOSDK](https://www.steinberg.net/asiosdk), open the top level folder in the .zip file and copy the contents into `[\path\to\jamulus\source]\libs\ASIOSDK2` so that, e.g., the folder `[\path\to\jamulus\source]\libs\ASIOSDK2\common` exists
```

Comment:
Isn't it easier if we make the folder libs/ASIOSDK2 already exist, so we can just say:

Download the [ASIOSDK](https://www.steinberg.net/asiosdk), open the top level folder in the .zip file and copy the contents into `[\path\to\jamulus\source]\libs\ASIOSDK2` so that, e.g., the folder `[\path\to\jamulus\source]\libs\ASIOSDK2\common` exists

Reply:
Copying the contents is more work than just extracting it – at least I think that's the case from CLI/Autobuild.

Reply:
The document says "create a folder" and "copy the contents of the archive so that..."
I think copying to an existing (empty) folder decreases the chance on mistakes.

Reply:
Hmm. I now tested what the standard Windows 11 behaviour actually is. Right clicking and starting extract gives a folder in a folder – that's indeed confusing. I'd like not to change the repo but better the description since the deploy_windows script should download the SDK and might need to be modified.

Reply:
> since the deploy_windows script should download the SDK and might need to be modified.

I wasn't aware of that. Isn't that against all ASIO rules as discussed in [this](https://github.com/jamulussoftware/jamulus/issues/2417#issuecomment-1068947074) issue)

If it is OK to automatically download it in the the deploy_windows script it could just as well be downloaded as a submodule like oboe.

Reply:
>I wasn't aware of that. Isn't that against all ASIO rules as discussed in https://github.com/jamulussoftware/jamulus/issues/2417#issuecomment-1068947074 issue.

I think "might need to be modified" was related to the build script, not to ASIO SDK files.

> If it is OK to automatically download it in the the deploy_windows script it could just as well be downloaded as a submodule like oboe.

Submodules only work for git repositories. ASIO SDK is only provided as a ZIP download.

Reply:
> Submodules only work for git repositories. ASIO SDK is only provided as a ZIP download.

OK not a submodule then, but it still could be downloaded from the make file. Either as a separate make target or as part of the build right ?

Reply:
I mean the deploy_windows.ps1 file already does that?